news/.gitignore

```
@@ -0,0 +1,3 @@
+venv
+.vscode
+__pycache__
```

```suggestion
**/__pycache__
```
Pycache directories will be created in all subdirectories when running the module

---

Subdirectories are already covered the way it is